jfrog-cli/utils/tests/utils.go

```diff
@@ -215,17 +219,24 @@ func NewJfrogCli(mainFunc func(), prefix, suffix string) *JfrogCli {
 func (cli *JfrogCli) Exec(args ...string) {
 	spaceSplit := " "
 	os.Args = strings.Split(cli.prefix, spaceSplit)
+	output := strings.Split(cli.prefix, spaceSplit)
 	for _, v := range args {
 		if v == "" {
 			continue
 		}
-		os.Args = append(os.Args, strings.Split(v, spaceSplit)...)
+		args := strings.Split(v, spaceSplit)
+		os.Args = append(os.Args, args...)
+		output = append(output, args...)
 	}
 	if cli.suffix != "" {
-		os.Args = append(os.Args, strings.Split(cli.suffix, spaceSplit)...)
+		args := strings.Split(cli.suffix, spaceSplit)
```

Comment:
cre suggest renaming *cli.suffix* to *cli.credentials* (because it only holds the credentials).
Let's also rename *cli.prefix* to *cli.args*.